src/schema/v2/homeView/sections/QuickLinks.ts

```diff
@@ -2,19 +2,37 @@ import { ContextModule, OwnerType } from "@artsy/cohesion"
 import { HomeViewSection } from "."
 import { HomeViewSectionTypeNames } from "../sectionTypes/names"
 import type { NavigationPill } from "../sectionTypes/NavigationPills"
+import { ResolverContext } from "types/graphql"
+import { getEigenVersionNumber, isAtLeastVersion } from "lib/semanticVersioning"
 
 export const QuickLinks: HomeViewSection = {
   id: "home-view-section-quick-links",
   contextModule: ContextModule.quickLinks,
   ownerType: OwnerType.quickLinks,
   type: HomeViewSectionTypeNames.HomeViewSectionNavigationPills,
   requiresAuthentication: true,
-  resolver: () => {
-    return QUICK_LINKS
+  resolver: (_parent, _args, context, _info) => {
+    return getDisplayableQuickLinks(context)
   },
 }
 
-const QUICK_LINKS: Array<NavigationPill> = [
+function getDisplayableQuickLinks(context: ResolverContext) {
+  return QUICK_LINKS.filter((quickLink) => {
+    let isDisplayable = true
+    const actualEigenVersion = getEigenVersionNumber(
+      context.userAgent as string
+    )
+    if (actualEigenVersion && quickLink.minimumEigenVersion) {
+      isDisplayable = isAtLeastVersion(
+        actualEigenVersion,
+        quickLink.minimumEigenVersion
+      )
+    }
+    return isDisplayable
+  })
```

Comment:
This is the gist of it.

Similar to the versioning we introduced at the home view section level, this logic considers all quick links displayable for _non_-Eigen clients.